src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-box search-box--big">
+      <input
+        class="search-txt search-txt--big"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
```

Comment:
Keep your attributes correctly formatted. The `<input>` tag starting on line 21 should have each attribute on a new line with proper indentation.